test/Microsoft.ML.Tests/OnnxTests.cs

```diff
@@ -268,6 +282,57 @@ public void KeyToVectorWithBagTest()
             Done();
         }
 
+        [Fact]
+        public void KmeansTest()
```

Comment:
>public void KmeansTest() [](start = 8, length = 24)

Have you tested this model loads in WinML?

Reply:
Yes! With the latest ONNXRUNTIME. Predictions just match.